### DISCUSSION TITLE
Can you show me an example on how to run it?

### DISCUSSION THREAD

**Discussion by RenderBlock** [meetDeveloper/freeDictionaryAPI#126] [Q&A]
I've been getting errors, can you show me exactly what to write and what code language to use for this?

**Comment by JWilkie1028** [comment]
What language are you using?
I can give you C# examples.

**Reply by igodacruz** [reply]
> What language are you using? I can give you C# examples.

I am trying to use this api in C# with RestSharp but in the response I can't deserialize the return at all.

The JSON appears in the content field of the response, but I can't transform it into a class to get the attributes of WORD, phonetics, audio and others.

Can you please make these examples available in C# to help me?
